src/components/Header/Tools.tsx

```diff
@@ -95,24 +116,59 @@ const Tools = () => {
   )}`;
 
   const enableAuthDropdownOption = useFlag('platform.chrome.dropdown.authfactor');
+  const enableExpandedSettings = true || useFlag('platform.chrome.expanded-settings');
```

Comment:
No need to bypass using feature flag.
```suggestion
  const enableExpandedSettings = useFlag('platform.chrome.expanded-settings');
```